mysite/blog/views.py

```diff
@@ -14,6 +15,32 @@ class PostListView(ListView):
     template_name = "blog/post/list.html"
 
 
+def add_comment(request, post_id):
```

Comment:
Since this view accepts only POST request, it is better to explicitly allow only POST requests here using https://docs.djangoproject.com/en/4.1/topics/http/decorators/#allowed-http-methods

Reply:
Updated the add comment to allow only POST requests as per suggestion
[https://github.com/Karthikeyantestpress/Django_Blog_Application/pull/10/files#diff-ae5986e281ef46bd77930a4985d5c376ce41689b3f9b85dcae0e6b60c77eecc8:~:text=%40require_http_methods(%5B,%2C%20post_id)%3A](url)